include/openmc/random_ray/source_region.h

```diff
@@ -96,6 +96,8 @@ class SourceRegion {
   OpenMPMutex lock_;
   double volume_ {0.0};
   double volume_t_ {0.0};
+  double volume_sq_ {0.0};
+  double volume_sq_t_ {0.0};
```

Comment:
Can you add some comments describing these data members (and all others while you're at it)? It's not immediately obvious to me what "volume_sq_" is supposed to mean, or the "t_" versions.

Reply:
I've added in some comments to the fields in this object to give an idea of what each one does. Generally, the `_sq` fields are the square of that field, and the `_t` fields are the accumulated totals over all iterations, rather than the iteration-specific value.